styles/base.less

```diff
@@ -302,4 +302,9 @@ atom-text-editor::shadow .highlight.current-result .region,
 atom-text-editor::shadow .highlight.current-result ~ .highlight.selection .region {
   background: lighten(@dark-gray, 20%); // to distinguish from the other results
   z-index: -1000;
-}
\ No newline at end of file
+}
+
+atom-text-editor::shadow  .bracket-matcher .region {
```

Comment:
I know, I'm just anal about these things. I review PRs all day, so it carries over to other projects :)